File: src/main/java/macro303/keschet/Colour.java
package macro303.keschet;

import org.jetbrains.annotations.Contract;
import org.jetbrains.annotations.NotNull;

/**
 * Created by Macro303 on 2018-02-08.
 */
public enum Colour {
	RESET("\u001B[0m"),
	RED("\u001B[31m"),
	GREEN("\u001B[32m"),
	YELLOW("\u001B[33m"),
	BLUE("\u001B[34m"),
	MAGENTA("\u001B[35m"),
	CYAN("\u001B[36m"),
	WHITE("\u001B[37m");

	@NotNull
	private final String colourCode;

	Colour(@NotNull String colourCode) {
		this.colourCode = colourCode;
	}

	@Contract(pure = true)
	@NotNull
	public String getColourCode() {
		return colourCode;
	}
}

File: src/main/kotlin/macro303/keschet/Colour.kt
package macro303.keschet

internal enum class Colour(val colourCode: String) {
	RESET(colourCode = "\u001B[0m"),
	RED(colourCode = "\u001B[31m"),
	GREEN(colourCode = "\u001B[32m"),
	YELLOW(colourCode = "\u001B[33m"),
	BLUE(colourCode = "\u001B[34m"),
	MAGENTA(colourCode = "\u001B[35m"),
	WHITE(colourCode = "\u001B[37m");
}